ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly.yaml

```diff
@@ -946,12 +946,7 @@ tests:
     cluster_profile: aws-qe
     env:
       CHANNEL_GROUP: nightly
-      E2E_RUN_TAGS: '@rosa'
       OPENSHIFT_VERSION: "4.14"
-      TEST_FILTERS: ~ChkUpgrade&;~NonPreRelease&;~Serial&;~Disruptive&;~DisconnectedOnly&;~HyperShiftMGMT&;~MicroShiftOnly&;ROSA&
-      TEST_FILTERS_SUPPLEMENTARY: ~ChkUpgrade&;~DisconnectedOnly&;~HyperShiftMGMT&;~MicroShiftOnly&;ROSA&
-    test:
-    - chain: openshift-e2e-test-qe
```

Comment:
why removing e2e tests? 
cc @xueli181114 @yasun1

Reply:
@yunjiang29   This test is just to install classic rosa and see if it is successful on the varying network bandwidth in the proxy, hence I am skipping e2e as the objective is achieved.

Reply:
@SachinNinganure  So the change will be roll back, right? Regularly, we will have the jobs with the stable version only do the healthy check. e.g. https://github.com/openshift/release/blob/903bef1840e8b72b3fba1c699b7957a103b42499/ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-stable.yaml#L217-L226

Reply:
yes..... infact I will close the pr itself once test runs are complete